src/tests/regress/data/sql/date.sql

```diff
@@ -223,22 +223,22 @@ SELECT f1 - date '2000-01-01' AS "Days From 2K" FROM DATE_TBL;
 --
 -- test extract!
 --
---@ SELECT f1 as "date",
---@     date_part('year', f1) AS year,
---@     date_part('month', f1) AS month,
---@     date_part('day', f1) AS day,
---@     date_part('quarter', f1) AS quarter,
---@     date_part('decade', f1) AS decade,
---@     date_part('century', f1) AS century,
---@     date_part('millennium', f1) AS millennium,
---@     date_part('isoyear', f1) AS isoyear,
---@     date_part('week', f1) AS week,
---@     date_part('dow', f1) AS dow,
---@     date_part('isodow', f1) AS isodow,
---@     date_part('doy', f1) AS doy,
---@     date_part('julian', f1) AS julian,
---@     date_part('epoch', f1) AS epoch
---@     FROM date_tbl;
+SELECT f1 as "date",
+    date_part('year', f1) AS year,
+    date_part('month', f1) AS month,
+    date_part('day', f1) AS day,
+    date_part('quarter', f1) AS quarter,
+    date_part('decade', f1) AS decade,
+    date_part('century', f1) AS century,
+    date_part('millennium', f1) AS millennium,
+    date_part('isoyear', f1) AS isoyear,
+    date_part('week', f1) AS week,
+    date_part('dow', f1) AS dow,
+    date_part('isodow', f1) AS isodow,
+    date_part('doy', f1) AS doy,
+    -- date_part('julian', f1) AS julian,
```

Comment:
The regress test runner is not smart enough for such change. We have 3 options here:
* implement `julian`
* keep the whole query disabled (`--@ `), and write a similar test in e2e slt
* update `expected/date.out` to comment this line as well, and remove the column in output.

Note that updating `expected/date.out` should be careful. It requires each nonempty line in `sql/date.sql` to have an exact match in it, after ignoring leading `--@ `. #6256
> Any extra lines in the output file are assumed to be a result of the preceding matching lines between input & output.

Reply:
I implemented `julian`.